src/optbuffer.jl

```diff
@@ -30,7 +30,8 @@ Base.@inline function Base.pop!(b::OptBuffer{T})::T where {T}
   val
 end
 
-
+Base.getindex(b::OptBuffer{T}, idx) where {T} = b.v[idx]
```

Comment:
@gkronber maybe `@inline` ?

Reply:
I started to refrain markin @inline because the Julia compiler should do so automatically.